tests/src/test/java/org/eclipse/hono/tests/amqp/AmqpAdapterTestBase.java

```diff
@@ -358,6 +359,7 @@ public void testUploadingXnumberOfMessages(final TestContext context) throws Int
 
         final Future<ProtonConnection> result = Future.future();
         final ProtonClientOptions options = new ProtonClientOptions();
+        options.addEnabledSaslMechanism(ProtonSaslPlainImpl.MECH_NAME);
```

Comment:
what about a test for a client using SASL_EXTERNAL?

Reply:
i want to add that later for a device that wants to authenticate using a client certificate. the current test code base does not use any certificate.. 

`SASL_PLAIN` is added mainly so that the current ITs passes. WDYT?

Reply:
I think that it would be good to have an integration test that uses SASL_EXTERNAL :-)

Reply:
BTW you should rebase so that the Travis build works again ...